WordPress/src/main/res/values/attrs.xml

```diff
@@ -107,4 +107,17 @@
         <attr name="wpTextDialogSubtitle" format="string" />
     </declare-styleable>
 
+    <!--
+        WPLoginInputRow
+    -->
+    <declare-styleable name="wpLoginInputRow">
+        <attr name="wpIconDrawable" format="reference"/>
+        <attr name="wpIconContentDescription" format="string"/>
+        <attr name="android:inputType" />
```

Comment:
Should the `wp` be removed from `wpIconDrawable` and `wpIconContentDescription` to be consistent with the other `wpLoginInputRow` attributes?